### ISSUE TITLE
Job is always executed immediately on yacron start

### ISSUE THREAD

**Issue by melkamar** [gjcarneiro/yacron#67]
* Yet Another Cron version: 0.16.0
* Python version: embedded
* Operating System: ubuntu20

### Description
The following job gets executed as soon as I start the `yacron` binary:
```yaml
jobs:
  - name: test-01
    command: echo "foobar"
    shell: /bin/bash
    schedule: "* * * * *"
```

But it should only be triggered every whole minute.

**Comment by andreas-wittig** [comment]
the same is true for {{schedule: "*/10 * * * *"}}.  Most propably the same issue.

**Comment by gjcarneiro** [comment]
fixed in 0.18
